recursion/flatten.js

```diff
@@ -0,0 +1,14 @@
+const flatten = (arr) => {
+    let flattenArray = [];
+    arr.forEach((item) => {
```

Comment:
do you think you can use [map](https://www.w3schools.com/jsref/jsref_map.asp) instead of forEach ? This could remove need of keeping variable flattenArray